src/Network/Send.pm

```diff
@@ -1221,4 +1221,42 @@ sub sendProduceMix {
 	debug "Sent Forge, Produce Item: $ID\n" , 2;
 }
 
+##
+# Request to merge item
+# 096E <size>.W { <index>.W }*
+# @author [Cydh]
+##
+sub sendMergeItemRequest {
+	my ($self, $num, $items) = @_;
+	my $len = ($num * 4) + 12;
+	$self->sendToServer($self->reconstruct({
+		switch => 'merge_item_request',
+		len => $len,
+		items => $items,
+	}));
+	debug "Sent merge item request: ".(join ', ', map {"$_->{itemIndex} x $_->{amount}"} @$items)."\n", "sendPacket";
+	undef @mergeItemList;
+}
+
+sub reconstruct_merge_item_request {
+	my ($self, $args) = @_;
+	$args->{itemList} = pack '(a2)*', map { pack 'v2', @{$_}{qw(itemIndex)} } @{$args->{items}};
+}
+
+##
+# Request to cancel merge item
+# 0974
+# @author [Cydh]
+##
+sub sendMergeItemCancel {
+	my ($self, $args) = @_;
+	$self->sendToServer($self->reconstruct({ switch => 'merge_item_cancel' }));
+	debug "Cancel Merge item\n", "sendPacket";
+	undef @mergeItemList;
+}
+
+sub reconstruct_merge_item_cancel {
+	my ($self, $args) = @_;
+}
```

Comment:
If any `parse`/`reconstruct` functions are empty, there's no need to include them at all. They're called only when they do exist:

https://github.com/OpenKore/openkore/blob/a668f7f8701974a995b51dfa9985631113bb8096/src/Network/PacketParser.pm#L185